AUTHORS.md

```diff
@@ -1,2 +1,3 @@
 - Peter Bourgon (@peterbourgon)
 - Tomás Senart (@tsenart)
+- Matt Moyer (@mattmoyer)
```

Comment:
Is this expected? Happy to drop this line or leave it.

Reply:
Yeah, this is capturing original authors. For contributors we rely on Github: https://github.com/oklog/ulid/graphs/contributors